example/ios/rnmshowcase/AppDelegate.mm

```diff
@@ -35,19 +31,4 @@ - (NSURL *)getBundleURL
 #endif
 }
 
-- (BOOL)turboModuleEnabled {
```

Comment:
I had to delete these booleans because it was causing crash on app init

Reply:
<img width="602" alt="Screenshot 2024-04-29 at 08 39 10" src="https://github.com/react-native-maps/react-native-maps/assets/37906388/b5e7927c-797c-43d0-9bee-83f40fb02248">

We should resolve this before merging since it's blocking us from using new arch on 0.74